h/static/scripts/group-forms/components/GroupModeration.tsx

```diff
@@ -103,8 +103,8 @@ function AnnotationListContent({
   return (
     <>
       {annotations?.map(anno => (
-        <article key={anno.id} className="border rounded p-2">
-          {anno.text}
+        <article key={anno.id}>
+          <AnnotationCard annotation={anno} />
         </article>
```

Comment:
The `AnnotationCard` contains section-related elements like `<header>` and `<footer>` so always need to be rendered inside an `<article>` or other sectioning element. I would move the `<article>` into `AnnotationCard`.